### PR TITLE
Fixed all critical issues and (most) moderate ones

### PR DESCRIPTION
**Pull request by valentinyanakiev**

### CODE REVIEW

**Comment by valentinyanakiev** [comment]
I have done a smoke test and didn't encounter any issues browsing, changing user privileges, sending an update.

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "replace-special-characters": "^1.2.5",
     "rimraf": "^3.0.2",
     "rxjs": "^7.3.0",
-    "sqlite3": "^5.0.2",
+    "sqlite3": "^4.2.0",
```
Comment (techsmyth) [inline package.json]: Do we even need sqlite nowadays?

Comment (valentinyanakiev) [reply package.json]: Removed in 443557f76bd044a67aec07fadcdf76ac30e40f97

**Review by techsmyth — APPROVED** [review]
Installing nicely for me...

**Pull request merged**